content/docs/rkvst-basics/managing-access-to-an-asset-with-abac/index.md

```diff
@@ -138,6 +138,20 @@ You may also grant permissions to an [App Registration](https://docs.rkvst.com/d
 **Note:** This is different from adding `subjects` as a key in your `access_permissions`, for example, when adding an external Subject ID to an OBAC policy. The user attribute `subject` refers to the Client ID associated with an App Registration.
 {{< /note >}}
 
+Additionally, you may set permissions based on JSON Web Tokens (JWTs). To do so, you must include the prefix `jwt_` followed by the desired claim. For example, the `user_attributes` should contain the key `jwt_foo` to match on claim `foo`.
+
+ ```json
+  "access_permissions": [
+         {
+             "asset_attributes_read": ["arc_display_name", "arc_description", "arc_home_location_identity", "Length", "Weight"],
+             "user_attributes": [
+                {"or": ["jwt_foo=4e28571b-6bb5-45a5-97e1-b1ab8a71acb7-app1"]}
```

Comment:
I think the claim example here might be confusing. To make it more relatable to real-world use cases I would change it to something along the lines of `{"or": ["jwt_app_reg_role=Tracker"]}`

Reply:
change made